blocks/popup/popup.css

```
@@ -0,0 +1,41 @@
+.popup div[role="dialog"][aria-hidden="false"] {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

You could simplify with:
```suggestion
  inset: 0
```